templates/components/common/navigation-list-alternate.html

```diff
@@ -0,0 +1,8 @@
+{{#if children}}
+    <a class="navPages-action navPages-action-depth-max has-subMenu is-root" href="{{url}}" data-collapsible="navPages-{{id}}">
+        {{name}} <i class="icon navPages-action-moreIcon" aria-hidden="true"><svg><use xlink:href="#icon-chevron-down" /></svg></i>
+    </a>
+    {{> components/common/navigation-dropdown}}
+{{else}}
+    <a class="navPages-action" href="{{url}}">{{name}}</a>
+{{/if}}
```

Comment:
i tried configuring my editor to remove it but it's saved by stencil-cli and it removes the nl.  i think it's the jsonlint module:  https://github.com/bigcommerce/stencil-cli/blob/master/lib/json-lint.js#L8.  it gets rid of the nl.  should we add it back when writing to file?